docs/sovereign/cross-chain-execution.md

```diff
@@ -0,0 +1,38 @@
+# Introduction
+
+When we take a look at the blockchain industry, we observe a segregated ecosystem lacking cohesion, interoperability, teamwork. Many strive to reach the top independently. The vision lead to the Blockchain Revolution, knows as “Web3” — a new era of the internet that is user-centered, emphasizing data ownership and decentralized trust.
+
+Sovereign Chains will dismantle the barriers between isolated blockchain networks by allowing smart contracts to seamlessly interact across different Sovereign Chains and the main MultiversX chain.
+This cross-chain interoperability is crucial for fostering an environment where decentralized apps (dApps) can utilize functionalities or assets from across the ecosystem.
+
+## What is Cross-Chain Execution?
+
+Cross-Chain execution is the ability of smart contracts or decentralized applications on one blockchain to invoke actions on another blockchain. This feature allows for seamless communication and interaction between different blockchain networks, enabling developers to build applications that are chain agnostic.
+
+
+## Cross-Chain Execution within Sovereign Chains
+
+This feature is enabled by using multiple smart contracts, each one with its unique role and set of functionalities. The current Sovereign Chain suite consists of three main contracts, here is the high-level description for some of the cross chain smart contracts:
+
+#### ESDT-Safe
+The *ESDT-Safe* Smart Contract performs all the heavy lifting. This is the cross-chain execution contract, this execution will be done between either any Sovereign Chain and the MultiversX mainchain or between any Sovereign Chains.
```

Comment:
- "cross-chain execution contract" - the contract does not execute anything cross-chain. It's responsible for the cross-chain transfers
- "between any Sovereign Chains" - not true